controllers/controller.js

```diff
@@ -10,4 +10,12 @@ res.status(200).send(topics.rows)
 });
 }
 
-module.exports = getAllTopics
\ No newline at end of file
+function getAllEndpoints(req, res, next){
+    selectAllEndpoints().then((response)=>{
+        res.set('Content-Type', 'application/json').status(200).send(response)
```

Comment:
You shouldn't necessarily need to use `res.set` here, `res.status(200).send()` should do.